src/ports/storage/component.ts

```diff
@@ -37,9 +38,38 @@ export function createStorageComponent(): IStorageComponent {
     return requestIdsBySocketId[socketId] ?? null
   }
 
+  const getIdentityId = (identityId: string) => {
```

Comment:
As we're storing and retrieving identities instead of identity ids, WDYT about renaming the functions as:

```suggestion
  const getIdentity = (identityId: string) => {
```